docs/content/contributing/terraform/prerequisites.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Prerequisites
+geekdocNav: true
+geekdocAlign: left
+geekdocAnchor: true
+---
+
+{{< toc >}}
+
+<!--
+NOTE:
+
+Removed ## Recommended Learning since advanced (not basic) knowledge is required around Git, TF and Docker, it is considered a hard requirement. Someone just looking at the Terraform Docs or code and trying to figure out how to contribute will not be successful. Contributors need to have a good understanding and holistic view of all tools and how to use them.
+
+-->
+
+## Tooling
+
+### Required Tooling
+
+{{< hint type=tip >}}
+
+We **strongly** recommend you use Linux or MacOS for your development environment. You can use Windows Subsystem for Linux (WSL) if you are using Windows.
+
+{{< /hint >}}
+
+To contribute to this project the following tooling is required:
+
+- [Git](https://git-scm.com/downloads)
+
+  If just installed, don't forget to set both your git username & password
+
+  ```PowerShell
+  git config --global user.name "John Doe"
+  git config --global user.email "johndoe@example.com"
+  ```
+
+- [Terraform](https://developer.hashicorp.com/terraform/downloads?product_intent=terraform)
+
+  {{< hint type=note >}}
+
+  Must be manually kept up-to-date.
+
+  {{< /hint >}}
+
+- [Visual Studio Code](https://code.visualstudio.com/download)
+  - [Terraform extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=hashicorp.terraform)
+  - [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)
+
+<br>
+
+---
+
+<br>
+
+### Recommended Tooling
+
+The following tooling/extensions are recommended to assist you developing for the project:
+
+#### Visual Studio Code Extensions
+
+- [CodeTour extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour)
```

Comment:
I think this should be removed

Reply:
What to be defined as prerequisites depends on what we recommend for testing/developing a module. I think there is an open space between using the container image `azterraform` (with all tools bootstrapped) and a contributor's local machine which has all predefined tools installed.

I'd like to discuss this point in our planned call so I can properly define the documentation around it. Marking this as `TBD`.

Reply:
Removed.